packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -137,7 +166,11 @@ export default class UIForm extends React.Component {
 						/>
 					))
 				}
-				<button type="submit" className="btn btn-primary">Submit</button>
+				<Buttons
+					id={`${this.props.id}-${this.props.formName}-actions`}
+					onTrigger={this.onTrigger}
```

Comment:
This is passed to each button to perform a trigger depending of its type and uiSchema configuration